### ISSUE TITLE
Please use .ruby_version when available

### ISSUE THREAD

**Issue by rosenfeld** [capistrano/rbenv#61]
I know there are other closed issues with regards to this subject, but I have a situation where I think it justifies changing this plugin to look first for a `.ruby_version` file.

In our Capistrano recipes we first stop the sidekiq worker before proceeding with the deploy. Now, consider when we are upgrading Ruby from 2.2.4 to 2.3.0 with rbenv. The `capistrano-sidekiq` gem will run "bundle exec sidekiqctl stop..." before the deploy starts. By that time, `.ruby_version` points to 2.2.4 and `bundle install` was run previously for the current path and it should just work. But since `capistrano-rbenv` is overriding the Ruby version to 2.3.0 this command will fail because `bundle install` wasn't executed yet before trying to stop sidekiq and it will yield to an error which I had to fix manually by running bundle in that path first for the newer Ruby.

If the project `.ruby_version` was used when present it should just work transparently...

**Comment by mattbrictson** [comment]
This may not work in your case, but have you considered enabling rbenv for non-interactive shells on your server and dropping capistrano-rbenv altogether?

For example, in my deployments where I am using Ubuntu 14.04 and have rbenv installed in `$HOME/.rbenv`, I just add this to the *very top* (i.e. preceding the interactive shell check) of my deployer user's `.bashrc`:

```
if [ -d $HOME/.rbenv ]; then
  export PATH="$HOME/.rbenv/bin:$PATH"
  eval "$(rbenv init -)"
fi
```

Then all my Capistrano deployments seamlessly use rbenv, and I get the automatic `.ruby_version` behavior like you described. And I haven't needed the capistrano-rbenv gem at all.

**Comment by rosenfeld** [comment]
@mattbrictson indeed, this is something I may consider doing. I measure the performance for setting up rbenv and it only takes about 2ms, which is reasonable enough I guess as a payload for the other commands not relying on rbenv...

**Comment by wojtha** [comment]
@rosenfeld What about this?

```ruby
set :rbenv_ruby, File.read('.ruby-version')
```

**Comment by rosenfeld** [comment]
This only works when Capistrano is part of the application's repository. In our case, we use Capistrano in a separate project, integrated with Chef to handle both server configuration and deploy with a single command. This project handles different applications possibly running on different Ruby versions. That's why this solution doesn't work for us.

**Comment by spikeheap** [comment]
Hi, we came up against this issue even with Capistrano being part of the application's repository.

To reproduce:

1. Create a task which uses `bundle`, and trigger it `after "deploy:starting"`.
2. Run a deploy.

Expected behaviour

**Comment by spikeheap** [comment]
Sorry for the pre-completion comment, updated the above to make more sense!

**Comment by mattbrictson** [comment]
Closed via #67

**Comment by mattbrictson** [comment]
Released in [v2.1.0](https://rubygems.org/gems/capistrano-rbenv/versions/2.1.0) 🚢
